src/app/containers/MostRead/Canonical/index.jsx

```diff
@@ -51,16 +64,45 @@ const CanonicalMostRead = ({ endpoint }) => {
     fetchMostReadData(endpoint);
   }, [endpoint, numberOfItems, datetimeLocale, lastUpdated, script, service]);
 
-  return items.length ? (
-    <MostRead
-      items={items}
-      header={header}
-      service={service}
-      script={script}
-      dir={dir}
-      labelId="Most-Read"
-    />
-  ) : null;
+  return (
+    <>
```

Comment:
Do you need this fragment?

Reply:
I just tried to remove it, seems like I need the fragment there in order to write the JS logic within JSX.

![image](https://user-images.githubusercontent.com/30599794/74232796-6aa7c380-4cc1-11ea-8b97-f28dcbd2b7fd.png)

Reply:
had to remove curly braces, cheers Tom